x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -354,7 +358,15 @@ private String randomDateForRange(long start, long end) {
         return DATE_FORMATTER.formatMillis(randomLongBetween(start, end));
     }
 
-    private void cloneSourceIndex(String sourceIndex, String sourceIndexClone) {
+    /**
+     * The source index is deleted at the end of the rollup process.
+     * We clone the source index, so that we validate rollup results against the
+     * source index clone.
+     *
+     * @return the name of the source index clone
+     */
+    private String cloneSourceIndex(String sourceIndex) {
+        String sourceIndexClone = "clone-" + sourceIndex;
```

Comment:
`sourceIndexClone` was an instance variable initialized at the test `setup()`. 

Now it's been generated here and this becomes more explicit